packages/cozy-realtime/src/RetryManager.spec.js

```diff
@@ -0,0 +1,152 @@
+import times from 'lodash/times'
+
+import RetryManager from './RetryManager'
+import { raiseErrorAfterAttempts, timeBeforeSuccessfull } from './config'
+
+const pause = time => new Promise(resolve => setTimeout(resolve, time))
```

Comment:
We usually use `sleep` name for this function.

Reply:
:+1: